core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -148,8 +158,11 @@ var TopicsAndSkillsDashboardPage = function() {
         createTopicButton,
         'Create Topic button takes too long to be clickable');
       createTopicButton.click();
-
       topicNameField.sendKeys(topicName);
+      topicDescriptionField.sendKeys(description);
+      // topicCategoryField.first().click();
```

Comment:
Never leave commented code in a PR. Please proofread your PR before sending for review.

Reply:
Sorry, fixed!